services/static-webserver/client/compile.json

```diff
@@ -56,7 +56,7 @@
       "class": "osparc.Application",
       "theme": "osparc.theme.products.s4l.ThemeDark",
       "name": "s4llite",
-      "title": "Sim4Life Lite",
+      "title": "Sim4Life.lite",
```

Comment:
Q: Can you instead use the `displayName` from `static-frontend-data.json`. This is taken from the product configuration.

Reply:
No. This is the configuration file for compiling the frontends.